### PR TITLE
Feature/inbox component

### PR DESCRIPTION
**Pull request by zhelezkov**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "sourceMap": true,
-    "noEmit": false
+    "noEmit": false,
+    "importsNotUsedAsValues": "error"
```
Comment (fsher) [inline tsconfig.json:21]: Good call! Better to use `import type` for types.

File: yarn.lock [hunk 2]
```diff
@@ -1004,20 +1004,6 @@
     js-sha256 "^0.9.0"
     tweetnacl "1.0.3"
 
-"@dialectlabs/web3@0.1.6":
```
Comment (fsher) [inline yarn.lock:1007]: Why was this removed from yarn.lock?

**Review by fsher — APPROVED** [review]
Great job! Please do double check whether we need to add any more components to the theme or not. 

Left a couple of minor comments and questions. Only one major one related to yarn.lock, please check.

File: packages/dialect-react-ui/components/Chat/index.tsx [hunk 3]
```diff
@@ -1,92 +1,47 @@
...
-  const toggleSettings = useCallback(
-    () => setSettingsOpen(!isSettingsOpen),
-    [isSettingsOpen, setSettingsOpen]
+export default function Chat({ inbox }: ChatProps): JSX.Element {
+  const { disconnectedFromChain, isWalletConnected, dialects } = useDialect();
+
+  const [activeRoute, setActiveRoute] = useState<Routes>(Routes.NoConnection);
+
+  useEffect(
+    function pickRoute() {
```
[the remark this answers is not in the document]
Comment (zhelezkov) [reply packages/dialect-react-ui/components/Chat/index.tsx:26]: Oh this is a good question actually. I've learned it at my previous job, while supporting an internal ui-kit library. 
It gives a handy effect name in react debugger, which might help to library users

Screenshot 2022-03-28 at 23 28 51

File: packages/dialect-react-ui/components/Chat/screens/Main/index.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+interface MainProps {
+  inbox?: boolean;
+}
+
+const Main = ({ inbox }: MainProps) => {
+  const { dialectAddress, dialects, setDialectAddress } = useDialect();
+
+  const { icons } = useTheme();
+
+  const [newThreadOpen, setNewThreadOpen] = useState<boolean>(false);
```
[the remark this answers is not in the document]
Comment (zhelezkov) [reply packages/dialect-react-ui/components/Chat/screens/Main/index.tsx:18]: explicit is better than implicit! :D

File: yarn.lock (hunk 2 above)
Comment (zhelezkov) [reply yarn.lock:1007]: I have updated everything to 0.1.11, so this version not used anymore

File: packages/dialect-react-ui/components/Chat/index.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (fsher) [reply packages/dialect-react-ui/components/Chat/index.tsx:26]: Ooooh, nice!

File: packages/dialect-react-ui/components/Chat/screens/Main/index.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (fsher) [reply packages/dialect-react-ui/components/Chat/screens/Main/index.tsx:18]: I agree with that, but with primitives it feels a bit unnecessary in the TS world, don't you think?

File: yarn.lock (hunk 2 above)
Comment (fsher) [reply yarn.lock:1007]: ah, perfect. Thanks!
